WalletWasabi.Tests/UnitTests/Crypto/ZeroKnowledge/LinearRelationTests.cs

```diff
@@ -159,31 +159,5 @@ public void StatementThrows()
 				new Equation(GroupElement.Infinity, new GroupElementVector(Generators.G, null!))
 			));
 		}
-
-		[Fact]
-		public void KnowledgeThrows()
```

Comment:
i meant to bring it up for discussion, not just do it yet, there are some downsides to this too and i'm not sure it's a good idea yet